### PR TITLE
Updated windows.sh

### PR DESCRIPTION
**Pull request by TheAggressive**
Added apt update before installing Ansible to fix a issue in windows.

### CODE REVIEW

File: windows.sh [hunk 1]
```diff
@@ -33,6 +33,7 @@ fi
 if [ ! -f /usr/bin/ansible ]; then
   echo "Installing pip..."
   sudo apt-get -y install python-pip
+  sudo apt-get -y update
```
Comment (swalkinshaw) [inline windows.sh]: Shouldn't this be before the line above it?:

```bash 
sudo apt-get -y update
sudo apt-get -y install python-pip
```

Comment (TheAggressive) [inline windows.sh]: It seems both ways work the same. I could change it to how you have it before installing pip.

Comment (swalkinshaw) [inline windows.sh]: Seems like it should be before. Are you testing from scratch to test? Anyway, let's put it before 👍 and we're good

Comment (TheAggressive) [inline windows.sh]: I am testing from scratch to verify it's working correctly. I just made that change so its before.

**Comment by swalkinshaw** [comment]
Thanks 👍

**Pull request merged**